pages/paperwork/edit.js

```diff
@@ -187,7 +187,7 @@ class Paperwork extends Component {
 
     this.dirtyFields.clear()
 
-    location = `/paperwork/${rescue.id}`
+    Router.push(`/paperwork/${rescue.id}`)
```

Comment:
This is how the redirects were done originally. The issue was that when we did the redirect, it would 404 originally, but in dev it would reload moments later when Next polled for changes. Can you verify that doesn't happen anymore?

Reply:
Just checked again and this behavior is now back. Not sure why...

Revising this.